src/components/Todos/TodoItem.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  return (
+    <div
+      data-cy="Todo"
+      className={classNames(
+        'todo',
+        { completed: todo.completed },
```

Comment:
if you do destructuration you can writ it down by shorter way
```suggestion
        { completed },
```